### ISSUE TITLE
Not really sync

### ISSUE THREAD

**Issue by fincha** [mark-hahn/markdown-scroll-sync#17]
I have the problem, I have a very big file, and if I search for a title, I see this on the left side, but not on the right. I have to scroll 2-3 screens to the the title.

**Comment by mark-hahn** [comment]
is there a big code section or image?  It cannot sync when they are on the
right.  It doesn't know what to do until it gets past the code or image.

**Comment by fincha** [comment]
this is an api documentation, so there are a lot of examples of code. no images.

**Comment by mark-hahn** [comment]
Can you see that the problem happens around blocks of code?  Or is it
something else?

**Comment by fincha** [comment]
I think so, the document use a lot of 

```
{
    "blabs": [
        {
            "id": 1,
            "name": "Host",
            "layer": "0",
            "blabs": "10793111",
            "top": "111",
            "nologin": false,
            "ipv4": "192.168.0.1/32",
            "blabs1": "unknown",
            "icon": "stub",
            "type": "Host",
            "left": "222",
            "mtu": "stub"
        },
	    [...]
}
```

I unfortunately I can send you the document

**Comment by mark-hahn** [comment]
The markdown preview is rendered as a complete read-only text editor.  The dom is very complex.  It would be very hard to match scrolling to the inside of code.  I will mark this as an enhancement.

**Comment by mark-hahn** [comment]
I've released a new version 2.0.1 that is a total rewrite.  I think it fixes all issues and works much better than before.  Please try it.
